docs/HARDWARE-VERIFICATION.md

```diff
@@ -0,0 +1,79 @@
+We must select the `<tag>` from `lfedge/eve-verification` dockerhub repository
+(i.e., `https://hub.docker.com/r/lfedge/eve-verification/tags`). The `<tag>`
+corresponds to the eve version we want to verify against the hardware.
+
+In cases such as using HPE iLO or Dell DRAC your BIOS will not be able to boot
+from a disk-based image, so we need an ISO image. The only difference is in
+step #1 that then becomes
+```docker run lfedge/eve-verification:<tag> verification_iso > verification.iso```.
+
+## Running the verification image via iPXE
+
+[iPXE](https://en.wikipedia.org/wiki/IPXE) is a modern Preboot eXecution
+Environment and a boot loader that allows operating system images to be
+downloaded right at the moment of booting (checkout [deployment](DEPLOYMENT.md)).
+To get the necessary images required by iPXE, we can execute the command
+```docker run lfedge/eve-verification:<tag> verification_net | tar xf -```.
+Apart from the image, iPXE expects a configuration file at certain URLs in
+order to proceed with the boot process. Here's an example of a iPXE
+configuration file used to run an EVE-OS verification image locally in a
+hardware lab. Note the changes made to both ```url``` and ```eve_args```
+variables.
+
+```console
+#!ipxe
+# set url https://github.com/lf-edge/eve/releases/download/snapshot/amd64.
+set url https://10.0.0.2/eve/releases/download/snapshot/amd64.
+# set eve_args eve_soft_serial=${ip} eve_reboot_after_install
+set eve_args eve_soft_serial=${ip} eve_install_server=zedcontrol.hummingbird.zededa.net eve_reboot_after_install
+
+# you are not expected to go below this line
+set console console=ttyS0 console=ttyS1 console=ttyS2 console=ttyAMA0 console=ttyAMA1 console=tty0
+set installer_args root=/initrd.image find_boot=netboot overlaytmpfs fastboot
+
+# you need to be this ^ tall to go beyond this point
+kernel ${url}kernel ${eve_args} ${installer_args} ${console} ${platform_tweaks} initrd=amd64.initrd.img initrd=amd64.verification.img initrd=amd64.initrd.bits initrd=amd64.rootfs.img
+initrd ${url}initrd.img
+initrd ${url}verification.img
+initrd ${url}initrd.bits
+initrd ${url}rootfs.img
+boot
+```
+
+## Logs of the verification process
+
+The verification images uses multiple utilities to gather the logs during the
+verification process. Among them we find: cpuinfo, meminfo, dmesg, smartctl,
+lsblk, lspci, lsusb, scsi, the hardware model as described in
+[HARDWARE-MODEL](./HARDWARE-MODEL.md), etc. Additionally, it stores the results
+of testing a static and a dynamic (i.e., using DHCP) network configuration and
+the read performance of the available storage devices using
+[fio](https://github.com/axboe/fio).
+
+## Getting the results of the verification process
+
+The verification image stores the results of all tests in the
+**invertory partition** of the boot media if possible (e.g., in the case of raw
+image). After we run the verification image, we can get the results of the
```

Comment:
From the documentation it is not clear to me from where is the user supposed to run `extract_verification_info.sh`.
Is it after running verification, unplugging USB from the verified device and plugging it back to the user's personal computer?
Is the user expected to get this script from github? Should we add example how to download this script?